synapse/lib/cell.py

```diff
@@ -82,6 +82,10 @@ async def __anit__(self, cell, link, user):
         self.user = user
         sess = self.link.get('sess')  # type: s_daemon.Sess
         sess.user = user
+        await self.initCellApi()
```

Comment:
Implements a cleaner/easier CellAPI startup hook.

Reply:
Have i told you how much i love this? because i love this.